backend/tests/buildings.test.js

```diff
@@ -2,45 +2,43 @@ const request = require('supertest');
 const app = require('../App');
 
 describe('Buildings', () => {
+  let token = null;
+
+  before((done) => {
+    request(app)
+      .post('/users/login')
+      .send({ username: 'testuser2', password: '12345678' })
+      .end((err, res) => {
+        token = res.body.token;
+        done();
+      });
+  });
```

Comment:
code duplication: this function could be in a separate `loginUser.js` file (or something similar naming) in the test folder, and be imported in every test file. this way if we delete testuser2 somehow, we can use another test user easily.